documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -0,0 +1,368 @@
+# Overview of Navigation/URL structure
+
+_This should give us a guide of what UI mockups we need to demonstrate, as well as inform the details of the Navigation links menu._
+
+## `/login`
+
+### Login Page
+
+When trying to access any URL, if not already authenticated, user is redirected to this page.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r2278](https://balsamiq.cloud/scs7giw/ponj59g/r2278)
+
+(Note: requested URL should be remembered, so that after successful login, user is redirected straight to that page)
+
+---
+
+## `/`
+
+### Home
+
+“Welcome” page showing a summary of activities in progress, e.g. incomplete applications, incomplete reviews, new notifications, etc.
+
+#### Applicant
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r53EF](https://balsamiq.cloud/scs7giw/ponj59g/r53EF)
+
+#### Reviewer
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r4BD1](https://balsamiq.cloud/scs7giw/ponj59g/r4BD1)
+
+---
+
+## `/applications`
+
+### Application list
+
+A list/table of all existing applications that the user is allowed to see.
+
+Clicking on an Application (a line in the list) takes you to the **Home/Overview** page for that application.
```

Comment:
```suggestion
Clicking on an Application (a line in the list) takes you to the **Home** page for that application.
```

I think we should start calling this one the Home page, since the previous called Home is now the Dashboard